### PR TITLE
Create skr.json

### PR DESCRIPTION
**Pull request by mpQadir**
Add Saraiki language

### CODE REVIEW

File: src/strings/skr.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
+{
+    "@metadata": {
+        "authors": [
+            "sushain97"
```
Comment (sushain97) [inline src/strings/skr.json:4]: update this?

File: src/strings/skr.json [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        ],
+        "last-updated": "2023-03-02",
+        "locale": [
+            "skr",
+            "skr"
```
Comment (sushain97) [inline src/strings/skr.json:9]: let's remove the duplicate here?

**Review by sushain97 — CHANGES_REQUESTED** [review]
Thanks! Let's add skr to https://github.com/apertium/apertium-html-tools/blob/master/src/strings/locales.json and add the changes of running `make` in https://github.com/apertium/apertium-html-tools/tree/master/src/strings.

**Comment by mpQadir** [comment]
Please, fix, remove duplicate and merge.

**Comment by sushain97** [comment]
@mpQadir is there someone in the community that can vouch for these translations? I unfortunately cannot properly review them.

**Comment by sushain97** [comment]
https://github.com/apertium/apertium-html-tools/pull/460 should be combined into this PR.

**Review by mpQadir — COMMENTED** [review]
This be done.  It is Ok.

**Comment by meesumalam** [comment]
> > @mpQadir is there someone in the community that can vouch for these translations? I unfortunately cannot properly review them.
> 
> @meesumalam could you take a look at these?

Yes, I can.

**Comment by mpQadir** [comment]
> @SaudQadir may verify these.



> > > @mpQadir is there someone in the community that can vouch for these translations? I unfortunately cannot properly review them.
> > 
> > 
> > @meesumalam could you take a look at these?
> 
> Yes, I can.

**Comment by SaudQadir** [comment]
> > @SaudQadir may verify these.
> 
> > > > @mpQadir is there someone in the community that can vouch for these translations? I unfortunately cannot properly review them.
> > > 
> > > 
> > > @meesumalam could you take a look at these?
> > 
> > 
> > Yes, I can.

I am Saraiki native, and I verify that the above translation is correct. and recommended for merge, please.

**Review by sushain97 — APPROVED** [review]

**Pull request merged**